packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx

```diff
@@ -21,3 +21,29 @@ Design tokens can now be found on the [Forma 36 documentation website](https://f
 ## Components
 
 React components available for immediate use.
+
+### Library Usage
+
+Install the package to your project
+
+```bash
+yarn add @contentful/forma-36-react-components
+```
+
+Or for NPM
+
+```bash
+npm i @contentful/forma-36-react-components
+```
+
+##### Import desired component into your project
+
+```jsx
+import { Input } from '@contentful/forma-36-react-components';
+```
+
+##### Import styles
+
+```js
+import '@contentful/forma-36-react-components/dist/styles.css';
+```
```

Comment:
Wow! I didn't know we had to do this 👀 
Do we do that in the webapp?

Reply:
yeah 🤯, we have it in the webpack config.